docs/sidebars.yaml

```diff
@@ -12,6 +12,10 @@
           page: commands/build-docs.md
         - label: bundle
           page: commands/bundle.md
+        - label: check-config
```

Comment:
Rebase one of these two docs pull requests - you've made this change in both of them

Reply:
I'll keep here, then rebase from main on the `translate` PR once this PR is merged.